atd-vzv/src/constants/popEsts.js

```diff
@@ -1,16 +1,19 @@
 export const popEsts = {
   years: {
-    2023: 1039855,
-    2022: 1025244,
-    2021: 1010835,
-    2020: 992208,
-    2019: 972499,
-    2018: 955094,
-    2017: 937065,
-    2016: 913917,
+    2026: 1040252,
+    2025: 1023982,
+    2024: 1007643, 
+    2023: 987508,
+    2022: 960915,
+    2021: 951989,
+    2020: 943549,
+    2019: 924887,
+    2018: 912041,
+    2017: 898677,
+    2016: 881422,
```

Comment:
what are the implications of adding future years here?  it's better to have some data here rather than no data, but with the understanding that the numbers will need to be updates as better data is available?

Reply:
adding 2024 now at least will put us in a better position than we were a year ago, where at the beginning of the year the vz viewer was broken without any estimate for the year. We will update as we get better data!